### ISSUE TITLE
Core assignment of `ActiveEffectData#origin` results in unintended UUIDs among synthetic actors

### ISSUE THREAD

**Issue by mkahvi** [foundryvtt/foundryvtt#9960]
### User Experience

Relative UUIDs are better when wanting to refer to current actor instead of different one, so Foundry should favor relative UUIDs when it doesn't specifically want another actor.

This mostly helps avoid unexpected behaviour with unlinked tokens that current behaviour can cause, since the AEs currently tend to point to the source actor instead of the synthetic one. Relative UUID would avoid the UUID becoming incorrect in such case.

This needs changing at least in:
- `Actors#fromCompendium()` - imported unlinked actor will have incorrect references once placed on scene.
- `Item#_onCreateDocuments()` - if the item is created on source actor, any unlinked actors inheriting it will behave weird with it.

This probably needs small changes to other code to support relative UUIDs, for example:
- `Item#_onDeleteDocuments` - needs to resolve relative UUIDs, but has questionable origin UUID handling in general

To summarize: use of relative origin UUIDs will fix rare issues with unlinked actors.

**Comment by aaclayton** [comment]
Good idea, but too late to make this change in V12. Shifting to V13.

**Comment by mkahvi** [comment]
I guess this missed v13, too? V14 maybe?

**Comment by stwlam** [comment]
Reframing: especially with the removal of `legacyTransferral` in V14, the meaning behind `origin` is fully system dependent and  needn't even refer to an Actor or Item. Addressing by removing the lone core assignment during creation of an ActiveEffect.
